design/fips-compliance.md

```diff
@@ -0,0 +1,199 @@
+# FIPS compliance for conjur-authn-k8s-client
+
+## Background
+
+The National Institute of Standards and Technology (NIST) issued the [FIPS
+ 140-2](https://csrc.nist.gov/publications/detail/fips/140/2/final) 
+Publication Series to coordinate the requirements and standards for cryptography 
+modules that include both hardware and software components.
+
+By making the conjur-authn-k8s-client FIPS compliant, our costumer will gain a 
```

Comment:
"customer"

Reply:
In this case i am actually referring to our people who wear costumes :)